desc/optimize/optimizer.py

```diff
@@ -198,7 +198,6 @@ def optimize(  # noqa: C901 - FIXME: simplify this
             eq_params_init = eq.params_dict.copy()
 
         options = {} if options is None else options
-        # TODO: document options
```

Comment:
this could have been nice

Reply:
I hope what I did works to add a link to the right page in the docs

Reply:
Thanks!